build/update-version.ps1

```diff
@@ -0,0 +1,8 @@
+$version = $args[0] -replace "-build", ""
```

Comment:
Should we give attribution to the inspiration of this script?  E.g., a link to repo.

Reply:
Good call - I'll add a link now